solidity/external/ethereum-sepolia/USDe.json

```diff
@@ -0,0 +1,3 @@
+{
+    "address": "0x07FcF6368d34D430aAFDc64435517842775fbF1B"
```

Comment:
[`0x6978E3e11b8Bc34ea836C1706fC742aC4Cb6b0Db` is the Mezo Portal](https://sepolia.etherscan.io/address/0x6978e3e11b8bc34ea836c1706fc742ac4cb6b0db) on Sepolia and the mock USDe address used is [`0xa166885a0b24ad149c8d66f3c845498655d93e74`](https://sepolia.etherscan.io/token/0xa166885a0b24ad149c8d66f3c845498655d93e74?a=0x6978e3e11b8bc34ea836c1706fc742ac4cb6b0db).

Reply:
Hmm, the address in the deployment is https://github.com/thesis/mezo-portal/blob/main/solidity/deployments/sepolia/USDe.json

Reply:
Okay, I now see there are two USDe in that contract.